docker-hub/repos/create.md

```diff
@@ -41,6 +41,39 @@ When creating a new repository:
   >
   > For organizations creating a new repository, it's recommended you select **Private**.
 
+### Add a repository overview
+
+Once you have created a repository, add an overview to the **Repository overview** field. This describes what your image does and how to use it.
+
+<div class="panel panel-default">
+  <div class="panel-heading collapsed" data-toggle="collapse" data-target="#collapseSample1" style="cursor: pointer">
+  Repository overview best practices
+  <i class="chevron fa fa-fw"></i></div>
+  <div class="collapse block" id="collapseSample1">
+    <p>A good image description is essential to help potential users understand why and how to use the image. The below covers the best practices to follow when adding a description to your image.</p>
+    <h3>Describe the image</h3>
```

Comment:
I'd make these h4. 

From the TOC it looks like they're all on one level, and when the accordion is closed I'm expecting to see these headers but don't.

![image](https://github.com/docker/docs/assets/102604716/daab6ca6-b2f0-48d4-9abe-d9e1a3b4e1ca)